core/src/components/select/test/select.spec.tsx

```diff
@@ -125,3 +125,35 @@ describe('select: slot interactivity', () => {
     expect(divSpy).toHaveBeenCalled();
   });
 });
+
+describe('ion-select: required', () => {
+  it('should have a aria-required attribute as true in inner button', async () => {
```

Comment:
```suggestion
  it('the inner button should have the aria-required attribute set to true', async () => {
```

Reply:
The current version aligns with the testing names standards for Ionic. Please leave it as is.